docs/src/content/docs/guides/components.mdx

```diff
@@ -169,6 +169,59 @@ import { LinkCard } from '@astrojs/starlight/components';
 	<LinkCard title="Components" href="/guides/components/" />
 </CardGrid>
 
+### Asides
+
+Asides (also known as “admonitions” or “callouts”) are useful for displaying secondary information alongside a page’s main content.
+
+An `<Aside>` can have an optional `type` of `note` (the default), `tip`, `caution` or `danger`. Setting a `title` attribute overrides the default aside title.
+
+````mdx
+# src/content/docs/example.mdx
+
+import { Aside } from '@astrojs/starlight/components';
+
+<Aside>A default aside without a custom title.</Aside>
+
+<Aside type="caution" title="Watch out!">
+	A warning aside *with* a custom title.
+</Aside>
+
+<Aside type="tip">
+    Other content is also supported in asides.
+
+    ```js
+    // A code snippet, for example.
+    ```
+
```

Comment:
I am guessing the extra line is auto added by Prettier?

Reply:
Yes, unfortunately :-(

Not sure if there are better Prettier rules we could configure? The MDX ones are honestly kind of odd.

Reply:
Played around in the Prettier playground a bit, but seems just to be how it is? Most of the time when things go wrong it even collapses the whole component children onto a single line 😬

Reply:
Another valid formatting option could be:

````mdx
<Aside type="tip">

Other content is also supported in asides.

```js
// A code snippet, for example.
```

</Aside>
````

This is at least balanced even if it takes some more space.